Create CONTRIBUTING.md (#10)
IntersectMBO/OSC-documentation: CONTRIBUTING.md

> # Contributing to [OSC-documentation]

Reviewer: ```suggestion
# Contributing to OSC-documentation
```

I would remove the `[ ]` because they would make it look like a hyperlink

Thank you for considering helping out! We're glad you're reading this because we need volunteer developers to help this project come to fruition. 
Looking to update information or feature your project? Feel free to make the change yourself.
First, please read the code of conduct. We take it very seriously!
Next, if you want to help out, do the following:
Fork the project.
Make a feature branch with the name of your change.
Make a change.
Commit your code.
Issue a Pull Request.
Once your PR is issued, we will review your work and decide to add it to the project!
For more details about contributing to GitHub projects, see How to use Github: Fork, Branch, Track, Squash, and Pull Request.

> ## Types of contribution

Reviewer: ```suggestion

## Types of contribution

```
spaces between headers are good practice with markdown files

Provide information on what kinds of contributions your project is looking for here. For example, these can be bug reports, help answering user questions, improving documentation, bug fixes, and new feature implementations.

## Openness

The configuration of an open project enables frictionless contributing. Projects should be discoverable and well-documented. Anyone within our Intersect MBO should be able to find a desired project and ramp up without direct guidance from host team members.

## Code of Conduct

> The Code of Conduct governs this project and everyone participating. By participating, you are expected to uphold this code. Please report unacceptable behavior to [email].

Reviewer: ```suggestion
[The Code of Conduct](https://github.com/IntersectMBO/OSC-documentation/blob/master/CODE-OF-CONDUCT.md) governs this project and everyone participating. By participating, you are expected to uphold this code. Please report unacceptable behavior to [oso@intersectmbo.org](https://github.com/IntersectMBO/committee-documentation-template/blob/main/oso@intersectmbo.org).
```

add link to code of conduct and oso email

## Reporting Bugs

> If you are subject to or witness unacceptable behavior or have any other concerns, please notify a community organizer as soon as possible.

Reviewer: this doesnt really make sense under Reporting Bugs - maybe a copy paste error?

## Contributing documentation

Include information on any documentation best practices your project follows, how to build documentation, checks to run, and how to submit the changes made back to the project.

## Transparency

For guest teams to be able to contribute meaningfully to a project, the host team must be transparent. This means that guest teams should be able to have an understanding of:

- The project/repo and its direction
- Outstanding feature requirements
- Progress on feature requirements
- Decision-making of the host team

## Prioritized Mentorship

Mentorship from the host team to the guest team via maintainers is crucial to Intersect MBO. Contributors on guest teams are up-leveled so that they understand enough about the host team’s project/repo to change it successfully.

## Voluntary Code Contribution

When contributing to this project, you must agree that you have authored 100% of the content, have the necessary rights to the content, and that the content you contribute may be provided under the project license.

## Your First Code Contribution

> Need help determining where to begin contributing to [Project Name]? You can start by looking through these `beginner` and `help-wanted` issues:

Reviewer: ```suggestion
Need help determining where to begin contributing to [OSC Documentation](https://github.com/IntersectMBO/OSC-documentation)? You can start by looking through these `beginner` and `help-wanted` issues:
```

- `Beginner` issues - issues that should only require a few lines of code and a test or two.
- `Help wanted` issues - issues which should be a bit more involved than `beginner` issues.

## Styleguides

Your work will be much appreciated if you follow these style guides in your contributions.

### Git Commit Messages

- Use the present tense ("Add feature," not "Added feature")
- Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
- Limit the first line to 72 characters or less
- Reference issues and pull requests liberally after the first line

### [Programming Language One] Styleguide

> All [Haskell] must adhere to [Haskell] Standard Style.

Reviewer: ```suggestion
```
can probably remove, as this repo wont have any haskell

### Documentation Styleguide

> Use Markdown for documentation.

Reviewer: ```suggestion
Use Markdown for documentation.
Following [best practices](https://www.markdownguide.org/basic-syntax/).
```
